### PR TITLE
done with grid component

### PR DESCRIPTION
**Pull request by Varun-28**
Done with grid components. It has the following types of the grid:

1. Grid with 2 items
2. Grid with 3 items
3. fully responsive and documented

grid-2
grid-3

### CODE REVIEW

**Review by Veekshith-Thoram — APPROVED** [review]

File: css/component_styles/grid.css [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+/* -------- grid css -------- */
+
+.grid-container {
+    display: grid;
+    background-color: var(--bg-color);
+    width: 100%;
+}
+
+.grid-item {
+    border: solid 2px var(--color-dnd);
```
Comment (Ronak-Mutha) [inline css/component_styles/grid.css:12]: I hope you have used the same format for border at other places also, for consistency

File: css/component_styles/grid.css [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+    padding: 1rem;
```
Comment (i-am-a-codegirl) [inline css/component_styles/grid.css:13]: It is suggested, that always use padding margin in multiples of 4 or 8 like padding you can give here 1.2 rem. It is a standard way of handling lengths

File: css/component_styles/grid.css [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+.three-unequal-grid{
+    grid-template-columns: 20% 50% 30%;
+}
+
+/* docs css */
+.grid-2{
+    min-height: 25rem;
```
Comment (i-am-a-codegirl) [inline css/component_styles/grid.css:47]: same here

File: css/component_styles/grid.css [hunk 4]
```diff
@@ -1,3 +1,52 @@
 @import url('./components.css');
 
-/* -------- grid css -------- */
\ No newline at end of file
+/* -------- grid css -------- */
+
+.grid-container {
+    display: grid;
+    background-color: var(--bg-color);
+    width: 100%;
```
Comment (i-am-a-codegirl) [inline css/component_styles/grid.css:8]: I would suggest using rems as it will make achieving responsiveness easier

**Review by i-am-a-codegirl — APPROVED** [review]
Code looks good to merge, just added few suggestions have a look on them

**Review by Ronak-Mutha — COMMENTED** [review]
I really liked the way you named your variables and classes. Good to merge

File: css/component_styles/grid.css (hunk 2 above)
Comment (Varun-28) [reply css/component_styles/grid.css:13]: Thanks for the suggestion I will follow this ✅

File: css/component_styles/grid.css (hunk 4 above)
Comment (Varun-28) [reply css/component_styles/grid.css:8]: percents(%) can also be used for achieving responsive so I use both rems and % as per need

File: css/component_styles/grid.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/grid.css:12]: Thanks for the review and yes I have used the same format everywhere ✅

**Pull request merged**